V2.16-RUN-15152-alertmanager-alerts (#633)
run-ai/docs: docs/admin/troubleshooting/alertmanager/RunaiDaemonSetRolloutStuck.md

---
title: RunaiDaemonSetRolloutStuck
summary: This article describes the RunaiDaemonSetRolloutStuck alert.
authors:
    - Jason Novich
    - Viktor Koukouliev
date: 2024-Jan-10
---

## Meaning

> The Run:ai daemonset has zero available pods on a relevant node.

Reviewer: ```suggestion
One or more of the Run:ai daemonsets has zero available pods on a relevant node.
```

## Impact

> Fractional gpu workloads support is not configured.

Reviewer: ```suggestion
Fractional GPU workloads may not be working as expected.
```

## Severity

Critical

## Diagnosis

Run

`kubectl get daemonset -n runai-backend`

> One or more of the daemonsets will have no running pods on certain nodes

Reviewer: ```suggestion
Identify the one or more of the daemonsets that have no running pods on some of the nodes.
```

## Mitigation

> Run `kubectl describe daemonset X -n runai` to try and figure out why the daemonset cannot create pods. Contact Run:ai if you cannot correct the issue.

Reviewer: ```suggestion
Run `kubectl describe daemonset X -n runai` on the relevant deamonset(s) to try and figure out why the it cannot create pods. 

If you cannot correct the issue, contact Run:ai support. 
```

Author: ```suggestion
Run `kubectl describe daemonset X -n runai` on the relevant deamonset(s) to try and figure out why it cannot create pods. 
If you cannot correct the issue, contact Run:ai support. 
```
